missile.py

```diff
@@ -1,14 +1,11 @@
-import pygame
-import math
+from constants import *
 
-class Missile:
+class Missile(pygame.sprite.Sprite):
     def __init__(self, x, y, mouse_x, mouse_y):
-        #location
-        self.x = x
-        self.y = y
-        #mouse clicks
-        self.mouse_x = mouse_x
-        self.mouse_y = mouse_y
```

Comment:
I can now see why you wanted me to review this PR. This will definitely conflict with the changes I've made to the missile class as well. We will get it figured out though